helm/loginapp/templates/deployment.yaml

```diff
@@ -81,6 +85,16 @@ spec:
       - name: config
         configMap:
           name: {{ include "loginapp.fullname" . }}
+      {{- if and .Values.templates.error .Values.templates.token }}
+      - name: templates
+        configMap:
+          name: {{ include "loginapp.fullname" . }}-templates
+          items:
```

Comment:
* conditional include, or a loop over all `templates` keys
* should support multi-line (`|-` & `| nindent 4`)